.github/workflows/ci.yml

```diff
@@ -44,12 +44,16 @@ jobs:
       libraries: |
         apt:
           - libopenjp2-7
+          - libhdf5-dev
         brew:
           - openjpeg
       envs: |
         - macos: py38
         - windows: py39
         - linux: py38-oldestdeps
+        # This job should be put back into cron when we can have a non-devdeps
+        # 3.11 job
```

Comment:
```suggestion
        # When a 3.11 compatible release of reproject is made the devdeps job in cron should be re-enabled
        # and this job should be changed to a normal non-devdeps job.
```

Reply:
To be a bit more explicit

Reply:
It's probably not just reproject (although it definitely *is* reproject). I have just tagged a reproject bugfix release with 3.11 support, so we can find out shortly lol.

Reply:
I have now removed this comment after a release of reproject.